pkg/chain/ethereum/tbtc/gen/Makefile

```diff
@@ -35,8 +36,13 @@ endef
 # See explanation in https://github.com/keep-network/keep-common/issues/117.
 define after_contract_hook
 	$(eval type := $(1))
+	$(if $(filter $(type),WalletProposalValidator),$(call fix_wallet_proposal_validator_contract_collision))
 	$(if $(filter $(type),MaintainerProxy),$(call fix_maintainer_proxy_contract_collision))
 endef
+define fix_wallet_proposal_validator_contract_collision
```

Comment:
In addition to updating `fix_wallet_proposal_validator_collision` a new function had to be added `fix_wallet_proposal_validator_contract_collision` which updates the generated `/contract/WalletProposalValidator.go` and `and /cmd/WalletProposalValidator.go`
The contract and cmd files did not have to updated this way when we only had a collision in `BitcoinTx.Info` as this type is wrapped with another type.
This is not true in case of `BitcoinTx.UTXO` which is passed directly into one of the contract's function.